server/historian/pnpm-lock.yaml

```diff
@@ -183,7 +183,7 @@ importers:
       '@types/ioredis': ^4.22.0
       '@types/mocha': ^10.0.0
       '@types/nconf': 0.0.37
-      '@types/node': ^14.18.0
+      '@types/node': ^16.11.7
```

Comment:
Curious: `@types/node` above shows as `16.18.16`. I guess this file was automatically updated, but is it a concern to have those different versions here?

Reply:
Good question! I think it is unimportant. The `^` means that it can be resolved to anything >= `16` and < `17`. It looks like `^16.11.7` actually resolved to `16.18.16` in terms of what was actually installed. I'll quickly find/replace to just line those up.